### DISCUSSION TITLE
There are breaks in some areas of the segmentation results output by the network

### DISCUSSION THREAD

**Discussion by Archaon6734** [Project-MONAI/MONAI#4628] [Q&A] [answered]
![微信截图_20220703133029](https://user-images.githubusercontent.com/58738002/177026451-e65e4959-e37f-445e-9d49-6e01af8cb191.png)
![微信截图_20220703132551](https://user-images.githubusercontent.com/58738002/177026450-1cde5c3c-8b37-4769-9c36-323edf639df2.png) During our research, we found that no matter what network is used, there will be breaks in some areas of the segmentation results. Why does this happen? Is there any related research work that we can refer to?

**Comment by yilei-wu** [comment, marked as answer]
the segmentation of these narrow tube structures is supposed to be hard and it's reasonable that the raw network output has these "breaks". I'm not sure what modality this is but I guess the intensity contrast on these structures is not apparent and the boundary is blurred. You can try some post-process techniques (CRF, level-set etc) to improve it). 
Also I think this is really NOT the responsibility of the developers here to do the analysis for you or find you the reference paper ( especially when the given information is so limited)

**Comment by wyli** [comment]
Agree with @yilei-wu, it'll be difficult to provide concrete suggestions in this case. If there are assumptions about the connectivities, it might be interesting to incorporate these priors during the model training (eg learning based CRF).
